### PR TITLE
location-proof-protocol-comments-AST-187

### PR DESCRIPTION
**Pull request by ronerlih**

### CODE REVIEW

File: location-proof-protocol.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#########################
+
+## For review only
+
+#########################
+
+# 📍 Location Proof Protocol
+
+The Location Proof Protocol aims to address the challenge of proving location on the decentralized web in a flexible, yet standardized way.
```
Comment (ronerlih) [inline location-proof-protocol.md:9]:
```suggestion
The Location Proof Protocol aims to address the challenge of proving location on the web, using the decentralized web in a flexible, yet standardized way.
```
..along those lines, to laminate that the problem is not only on web3, but it's a general problem on the internet, still, maybe just distracts focus, whatever is best.

File: location-proof-protocol.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Given the diverse requirements and approaches for location proofs, our goal is to support innovation across a wide range of proof strategies
+while providing a standard data model for creating and verifying these proofs.
```
Comment (ronerlih) [inline location-proof-protocol.md:12]:
```suggestion
Given the diverse requirements and approaches for location proofs, our goal is to support innovation across a wide range of proof strategies, while providing a standard data model for creating and verifying these proofs.
```

File: location-proof-protocol.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+This repository currently contains the code necessary to deploy the core Ethereum Attestation Service (EAS) schema for the protocol. The
+standard includes a framework (in development) for building "recipes" that allow for many different strategies for proving location to be
+added.
+
+## The Challenge
+
+### Broad Approaches to Location Proofs
```
Comment (ronerlih) [inline location-proof-protocol.md:20]:
```suggestion
### Broad classifications of Location Proofs
```
?

File: location-proof-protocol.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+- **Authority-based Strategies**: Authorized individuals attest to someone's presence. i.e. ticket checkers, event hosts
+- **Social Strategies**: People verify each other's claims, confirming or challenging their validity. i.e. permissionless social
+  confirmation, perhaps with sybil-resistant scoring of confirmer accounts; PIN exchange
+- **Near-field Machine Strategies**: Hardware devices attest to the presence of another device through information exchange or cryptographic
+  'endorsement' of a claim. i.e. RFID; NFC; Bluetooth
+- **Network Machine Strategies**: Nodes on a local network or hardware wallets triangulate position and sign confirmation attestations. i.e.
+  Time of Flight; Time Difference of Arrival
+- **Sensor Data Strategies**: Where devices sense local networks, environmental conditions, images + audio, accelerometers, etc, and
+  location is discerned from analysis of this evidence. i.e. radio frequency / optical / inertial / acoustic / magnetometer localization
```
Comment (ronerlih) [inline location-proof-protocol.md:30]: Should this be combined with "Near-field"

File: location-proof-protocol.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+In addition to these different categories of strategies, various verification schemes can be employed to endow location proof recipes with a
+different attributes, such as privacy, game theoretic assurances of validity, etc.
+
+- **Time-based Schemes**: Involves waiting or challenge periods.
+- **Cryptographic Schemes**: Zero-knowledge proofs verified on-chain or in peer nodes.
+- **AI-based Schemes**: Analyzing photos or contextual information.
+- **Economic Schemes**: Requires payment or staking value for proof creation.
+
```
Comment (ronerlih) [inline location-proof-protocol.md:39]: Maybe an addition, along the lines of, 
we plan to support these schemas, and others created according to need, with the same recipe interface to adhear to, and to be unoppinionated about the payload (`Array<bytes>`) for the verifier method.
(maybe to details for this context).

File: location-proof-protocol.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- **Sensor Data Strategies**: Where devices sense local networks, environmental conditions, images + audio, accelerometers, etc, and
+  location is discerned from analysis of this evidence. i.e. radio frequency / optical / inertial / acoustic / magnetometer localization
+
+In addition to these different categories of strategies, various verification schemes can be employed to endow location proof recipes with a
+different attributes, such as privacy, game theoretic assurances of validity, etc.
+
+- **Time-based Schemes**: Involves waiting or challenge periods.
+- **Cryptographic Schemes**: Zero-knowledge proofs verified on-chain or in peer nodes.
+- **AI-based Schemes**: Analyzing photos or contextual information.
+- **Economic Schemes**: Requires payment or staking value for proof creation.
```
Comment (ronerlih) [inline location-proof-protocol.md:38]: And perhaps with Adam's suggestion, a gasless view function to be called on need.

File: location-proof-protocol.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+Astral's Location Proof Protocol aims to support location proofs within a wide range of different spatial reference systems, both global and
+relative. However, initial development of the protocol will focus on positioning objects / events on Earth.
+
+## Our Approach
+
+### Generalized Location Proof Protocol
+
+We are developing a protocol that accommodates various proof strategies while maintaining a standard data model and tools for creating and
+verifying location proofs.
```
Comment (ronerlih) [inline location-proof-protocol.md:59]:
```suggestion
We are developing a protocol that accommodates various proof strategies while maintaining a standard data model and tools for creating and verifying location proofs.
```

Comment (ronerlih) [reply location-proof-protocol.md:59]: (probably prettier, then somehow I have other config.. tbd :)

File: location-proof-protocol.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+## Repository Contents
+
+- **Contract Deployment Scripts**: Scripts to deploy the core EAS schema for the Location Proof Protocol
```
Comment (ronerlih) [inline location-proof-protocol.md:63]: Maybe to setup a link to tree, or just links, or a `.md` table ? 
Or the table you set up on notion?
```
├── package.json
├── remappings.txt
├── script
│   ├── Base.s.sol
│   └── Deploy.s.sol
├── src
│   ├── contracts
│   └── interface
└── test
    └── RegistryFactory.t.sol
  ```

File: location-proof-protocol.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+- **Proof Creation Documentation**: Detailed documentation on how to interact with the EAS schema
+- **Location Proof Strategy Development Guides**: Detailed information on how to create and register recipes for creating location proofs
+  based on different strategies
+
+## Core Schema v0.1 (Sepolia Testnet)
+
+The current working schema described [here](./schema/README.md) is deployed on Ethereum Sepolia testnet, UID
```
Comment (ronerlih) [inline location-proof-protocol.md:70]: Maybe still best to add teh schema fields here

**Review by ronerlih — COMMENTED** [review]
✨  Great work @johnx25bd !!
I'd added comments, more in the spirit of thinking of this out laud. 
Approved as is from my side, for your consideration.

File: location-proof-protocol.md (hunk 4 above)
Comment (johnx25bd) [reply location-proof-protocol.md:30]: I see them as different — Near-field is an active scan of another device, sensor data is picking up ambient measurements of the physical and digital environment

**Pull request merged**